config/config.go

```diff
@@ -846,14 +847,16 @@ func (c *Config) parseHistogramBuckets() ([]float64, error) {
 
 func (c *Config) parseForwardedHeaders() error {
 	for _, header := range c.ForwardedHeadersList.values {
-		switch header {
-		case "X-Forwarded-For":
+		switch {
+		case header == "X-Forwarded-For":
 			c.ForwardedHeaders.For = true
-		case "X-Forwarded-Host":
+		case header == "X-Forwarded-Host":
 			c.ForwardedHeaders.Host = true
-		case "X-Forwarded-Proto=http":
+		case strings.HasPrefix(header, "X-Forwarded-Port="):
+			c.ForwardedHeaders.Port = strings.TrimPrefix(header, "X-Forwarded-Port=")
```

Comment:
hmm so it can be any kind of string?
Maybe better to check it's only a number of the valid port range (uint16)?

Reply:
I wrote that initially but then I think it does not worth it since there also will be a uint16->string conversion to the header value so I've though users are ably to provide correct value

Reply:
we could also just validate with strconv.Atoi if it returns error and if not we add the string to the port header. So no 2x conversion required.
For me the resasoning is also fine so I add my +1, decide yourself.